bx_py_utils/anonymize.py

```diff
@@ -36,3 +36,20 @@ def anonymize(value: str, handle_email: bool = True) -> str:
 
     value = f'{value[:1]}{value[1:-1].translate(_ANONYMIZATION_TRANS)}{value[-1:]}'
     return value
+
+
+def anonymize_dict(data: dict, keys=('secret', 'password', 'token')) -> dict:
```

Comment:
`keys` is a bit generic, can we call it `secret_keys` and make it a (frozen)set?

Reply:
done